lib/fb/post.rb

```diff
@@ -0,0 +1,34 @@
+# Ruby client to authenticate a Facebook user.
+# @see http://www.rubydoc.info/gems/Fb/
+module Fb
+  # Fb::Post reprensents a Facebook post. Post provides getters for:
+  #   :id, :title, :url, :created_time, :type, :length, and :insights.
+  class Post
+    attr_reader :id, :title, :url, :created_time, :type, :length, :insights
+
+    # @param [Hash] options the options to initialize an instance of Fb::Post.
+    # @option [String] :id The post id.
+    # @option [String] :title The status message in the post or post story.
+    # @option [String] :url URL to the permalink page of the post.
+    # @option [String] :created_time The time the post was initially published.
+    # @option [Hash] :insights a collection of metrics.
+    # @option [String] :type A string indicating the object type of this post.
+    # @option [String] :length The length of the attached video.
+    def initialize(options = {})
+      @id = options["id"]
+      @title = options["message"] || options["story"]
```

Comment:
@S1v4 what is the difference between `message` and `story`?

Do some posts have a message and others have a story?